contrib/platform/src/com/sun/jna/platform/win32/PowrProf.java

```diff
@@ -0,0 +1,231 @@
+/* Copyright (c) 2019 Daniel Widdis, All Rights Reserved
+ *
+ * The contents of this file is dual-licensed under 2 
+ * alternative Open Source/Free licenses: LGPL 2.1 or later and 
+ * Apache License 2.0. (starting with JNA version 4.0.0).
+ * 
+ * You can freely decide which license you want to apply to 
+ * the project.
+ * 
+ * You may obtain a copy of the LGPL License at:
+ * 
+ * http://www.gnu.org/licenses/licenses.html
+ * 
+ * A copy is also included in the downloadable source code package
+ * containing JNA, in file "LGPL2.1".
+ * 
+ * You may obtain a copy of the Apache License at:
+ * 
+ * http://www.apache.org/licenses/
+ * 
+ * A copy is also included in the downloadable source code package
+ * containing JNA, in file "AL2.0".
+ */
+package com.sun.jna.platform.win32;
+
+import com.sun.jna.Library;
+import com.sun.jna.Native;
+import com.sun.jna.Pointer;
+
+/**
+ * Functions used with power management.
+ */
+public interface PowrProf extends Library {
+    PowrProf INSTANCE = Native.load("PowrProf", PowrProf.class);
+
+    /**
+     * Enum which indicates the power information level requested from
+     * {@link #CallNtPowerInformation()}. This value indicates the specific
+     * power information to be set or retrieved.
+     */
+    public interface POWER_INFORMATION_LEVEL {
```

Comment:
This enum is defined in WinNT.h. I can live with it in PowerProf, as to my observation it is only used there.

Reply:
Hm. I saw [here](https://docs.microsoft.com/en-us/windows-hardware/drivers/ddi/content/wdm/ne-wdm-power_information_level) that it was in `wdm.h` where in fact it is... seems it's defined twice.   This SO answer implies that the `wdm.h` file is for kernel-mode calls while the `winnt.h` version is for user-mode calls.

So it's probably correct to put it in `WinNT` for the `NtPowerInformation` variant, while the other one would work for the `ZwPowerInformation` variant.   Let me know if you'd like me to move it.

Reply:
Another difference is that I only included the enum values relevant to this function call, which makes sense if we leave it where it is. If we move it I'd suggest filling in all the values.

Reply:
ok - will take it as is. Thank you for the work!